charts/admission-controller/templates/webhook/deployment.yaml

```diff
@@ -29,9 +29,12 @@ spec:
                   matchLabels:
                     {{- include "admissionController.webhook.selectorLabels" . | nindent 20 }}
               weight: 10
-      {{- with .Values.webhook.imagePullSecrets }}
+      {{- if .Values.global.image.pullSecrets }}
```

Comment:
Wouldn't make sense to invert `if` and `else if` statements?
This way we can use `global.image.pullSecrets` as a default but give the user the ability to override it specifying the `webhook.imagePullSecrets`